lib/event_logger.ex

```diff
@@ -1,2 +1,31 @@
 defmodule EventLogger do
+  import Logger, only: [info: 1, warn: 1, error: 1]
+  import String, only: [downcase: 1]
+
+  def log(level, data) do
+    case level do
```

Comment:
instead of the case here, you could add a guard to the function like this:
```
def log(level, data) when level in [:error, :warn, :info] do

end
```

Reply:
Im not sure this would work as you would expect it to, as apply cannot be used for functions